build/illume.js

```diff
@@ -11,17 +11,15 @@ var _q = require("@standard-library/q");
 var _kefir = require("kefir");
 
 var offsetAbove = function offsetAbove(y, element) {
-  return y > $(element).offset().top;
+  return y > element.offsetTop;
```

Comment:
OK, I have added a test and have changed the JS! I couldn't figure out how to get a test to run all the browsers, so I just had it defaulting to Chrome for now